scripts/watch.ts

```diff
@@ -112,7 +112,7 @@ const main = async () => {
         // replace that and handle it here, but it doesn't seem
         // necessary to.
         onChange: async ({ spawn }) => {
-          await spawn`yarn turbo run dev`;
+          await spawn`yarn turbo run dev --filter=!./apps/anon-message-client --filter=!./apps/anon-message-client --filter=!./apps/kudosbot-client`;
```

Comment:
could we instead replace this with
```ts
await spawn`yarn turbo run dev:apps`;
```
?

Reply:
No, because then it won't run the dev task for any packages that need it (right now that's just `passport-ui`, but they do exist).